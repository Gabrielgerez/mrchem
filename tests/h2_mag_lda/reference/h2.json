{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_per_node": -1,
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000,
      "use_omp_num_threads": false
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "h2",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {
      "ext_mag-0.000000": {
        "components": [
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_0_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_0_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_0_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_0_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_0_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_0_vector.json",
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "file_x_a": "initial_guess/X_a_rsp_0",
              "file_x_b": "initial_guess/X_b_rsp_0",
              "file_x_p": "initial_guess/X_p_rsp_0",
              "file_y_a": "initial_guess/Y_a_rsp_0",
              "file_y_b": "initial_guess/Y_b_rsp_0",
              "file_y_p": "initial_guess/Y_p_rsp_0",
              "prec": 0.001,
              "type": "none"
            },
            "rsp_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "final_prec": 0.001,
              "helmholtz_prec": -1.0,
              "kain": 3,
              "max_iter": 10,
              "method": "DFT",
              "orbital_thrs": 0.01,
              "orth_prec": 1e-14,
              "property_thrs": -1.0,
              "start_prec": 0.001
            }
          },
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_1_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_1_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_1_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_1_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_1_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_1_vector.json",
              "file_chk_x": "checkpoint/X_rsp_1",
              "file_chk_y": "checkpoint/Y_rsp_1",
              "file_x_a": "initial_guess/X_a_rsp_1",
              "file_x_b": "initial_guess/X_b_rsp_1",
              "file_x_p": "initial_guess/X_p_rsp_1",
              "file_y_a": "initial_guess/Y_a_rsp_1",
              "file_y_b": "initial_guess/Y_b_rsp_1",
              "file_y_p": "initial_guess/Y_p_rsp_1",
              "prec": 0.001,
              "type": "none"
            }
          },
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_2_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_2_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_2_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_2_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_2_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_2_vector.json",
              "file_chk_x": "checkpoint/X_rsp_2",
              "file_chk_y": "checkpoint/Y_rsp_2",
              "file_x_a": "initial_guess/X_a_rsp_2",
              "file_x_b": "initial_guess/X_b_rsp_2",
              "file_x_p": "initial_guess/X_p_rsp_2",
              "file_y_a": "initial_guess/Y_a_rsp_2",
              "file_y_b": "initial_guess/Y_b_rsp_2",
              "file_y_p": "initial_guess/Y_p_rsp_2",
              "prec": 0.001,
              "type": "none"
            }
          }
        ],
        "dynamic": false,
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 0.001,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 0.001
          },
          "xc_operator": {
            "shared_memory": false,
            "xc_functional": {
              "cutoff": 0.0,
              "functionals": [
                {
                  "coef": 1.0,
                  "name": "lda"
                }
              ],
              "spin": true
            }
          }
        },
        "frequency": 0.0,
        "perturbation": {
          "derivative": "abgv_00",
          "operator": "h_b_dip",
          "r_O": [
            0.0,
            0.0,
            0.0
          ]
        },
        "properties": {
          "magnetizability": {
            "mag-0.000000": {
              "derivative": "abgv_00",
              "dia_operator": "h_bb_dia",
              "frequency": 0.0,
              "para_operator": "h_b_dip",
              "precision": 0.001,
              "r_O": [
                0.0,
                0.0,
                0.0
              ]
            }
          },
          "nmr_shielding": {
            "nmr-0h": {
              "derivative": "abgv_00",
              "dia_operator": "h_bm_dia",
              "para_operator": "h_m_pso",
              "precision": 0.001,
              "r_K": [
                0.0,
                0.0,
                -0.7001435291738478
              ],
              "r_O": [
                0.0,
                0.0,
                0.0
              ],
              "smoothing": 0.001
            }
          }
        },
        "unperturbed": {
          "fock_operator": {
            "coulomb_operator": {
              "poisson_prec": 0.001,
              "shared_memory": false
            },
            "exchange_operator": {
              "exchange_prec": -1.0,
              "poisson_prec": 0.001
            },
            "kinetic_operator": {
              "derivative": "abgv_55"
            },
            "nuclear_operator": {
              "nuclear_model": "point_like",
              "proj_prec": 0.001,
              "shared_memory": false,
              "smooth_prec": 0.001
            },
            "xc_operator": {
              "shared_memory": false,
              "xc_functional": {
                "cutoff": 0.0,
                "functionals": [
                  {
                    "coef": 1.0,
                    "name": "lda"
                  }
                ],
                "spin": true
              }
            }
          },
          "localize": false,
          "precision": 0.001
        }
      }
    },
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "lda"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT",
        "prec": 0.001,
        "relativity": "None",
        "restricted": false,
        "rotate": true,
        "screen": 12.0,
        "type": "gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        2.544077373851441e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 2.3125524059495336e-05,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            2.3125524059495336e-05
          ],
          "vector_el": [
            0.0,
            0.0,
            2.312552406982041e-05
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "magnetizability": {
        "mag-0.000000": {
          "frequency": 0.0,
          "isotropic_average": null,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "tensor": [
            -0.9131112520552285,
            0.0,
            0.0,
            null,
            null,
            null,
            null,
            null,
            null
          ],
          "tensor_dia": [
            -0.9131112520552285,
            0.0,
            0.0,
            0.0,
            -0.9131112520552256,
            0.0,
            0.0,
            0.0,
            -0.7722102648042812
          ],
          "tensor_para": [
            0.0,
            0.0,
            0.0,
            null,
            null,
            null,
            null,
            null,
            null
          ]
        }
      },
      "multiplicity": 1,
      "nmr_shielding": {
        "nmr-0h": {
          "anisotropy": null,
          "diagonalized_tensor": [
            null,
            null,
            null
          ],
          "isotropic_average": null,
          "r_K": [
            0.0,
            0.0,
            -0.7001435291738478
          ],
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "tensor": [
            24.969077211216337,
            0.0,
            0.0,
            null,
            null,
            null,
            null,
            null,
            null
          ],
          "tensor_dia": [
            24.969077211216337,
            0.0,
            0.0,
            0.0,
            24.969077211216423,
            0.0,
            0.0,
            0.0,
            27.417678846755102
          ],
          "tensor_para": [
            0.0,
            0.0,
            0.0,
            null,
            null,
            null,
            null,
            null,
            null
          ]
        }
      },
      "orbital_energies": {
        "energy": [
          -0.34528396828216973,
          -0.34528396828216973
        ],
        "occupation": [
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "b"
        ],
        "sum_occupied": -0.6905679365643395
      },
      "scf_energy": {
        "E_ee": 1.328363917961886,
        "E_eext": 0.0,
        "E_el": -1.8186520280666034,
        "E_en": -3.566631555859595,
        "E_kin": 1.0734860652323075,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -1.1045127420976426,
        "E_x": 0.0,
        "E_xc": -0.6538704554012021,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": {
      "ext_mag-0.000000": {
        "components": [
          {
            "rsp_solver": {
              "converged": true,
              "cycles": [
                {
                  "mo_residual": 0.08780898201594486,
                  "property_update": 0.0,
                  "symmetric_property": 0.0,
                  "wall_time": 2.233762939
                },
                {
                  "mo_residual": 0.022383880371404045,
                  "property_update": 0.0,
                  "symmetric_property": 0.0,
                  "wall_time": 2.880983301
                },
                {
                  "mo_residual": 0.0003591406832373802,
                  "property_update": 0.0,
                  "symmetric_property": 0.0,
                  "wall_time": 2.543173484
                }
              ],
              "wall_time": 7.657986357
            }
          },
          null,
          null
        ],
        "frequency": 0.0,
        "perturbation": "h_b_dip",
        "success": false
      }
    },
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.328363917961886,
        "E_eext": 0.0,
        "E_el": -1.8186520280666034,
        "E_en": -3.566631555859595,
        "E_kin": 1.0734860652323075,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -1.1045127420976426,
        "E_x": 0.0,
        "E_xc": -0.6538704554012021,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": false
  }
}
