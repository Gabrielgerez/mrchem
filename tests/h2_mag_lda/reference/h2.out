                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         text                                     ***
***         Git commit hash    20e53b0ebebbbed503cb-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Thu Feb 20 16:31:25 2025 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : merge
 Git commit hash       : d82d0d16fbefc341d12f
 Git commit author     : gitpeterwind
 Git commit date       : Thu Feb 20 16:02:13 2025 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project GTO molecular orbitals
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 MO alpha file           : initial_guess/mrchem.moa
 MO beta file            : initial_guess/mrchem.mob
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999041879e-01
    1    1    b          :                              9.999999041879e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.073486065232
 E-N energy              :            (au)                 -3.566631555860
 Coulomb energy          :            (au)                  1.328363917962
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.653870455401
 N-N energy              :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.818652028067
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.104512742098e+00
                         :      (kcal/mol)             -6.930922098896e+02
                         :        (kJ/mol)             -2.899897806178e+03
                         :            (eV)             -3.005532283883e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.345283968282
    1    1    b          :            (au)                 -0.345283968282
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.690567936564
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***               Computing Linear Response Wavefunction                ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              0.000000000000e+00
    1    1    b          :                              0.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize linear response orbitals
 Frequency               : Static field
 Perturbation            : h_B_dip[x]
 Method                  : DFT
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 10
 KAIN solver             : 3
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Property threshold      : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual       Symmetric property                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.000000000000          0.000000e+00
    1          8.780898e-02           0.000000000000          0.000000e+00
    2          2.238388e-02           0.000000000000          0.000000e+00
    3          3.591407e-04           0.000000000000          0.000000e+00
---------------------------------------------------------------------------
                      SCF converged in 3 iterations!
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              0.000000000000e+00
    1    1    b          :                              0.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              0.000000000000e+00
    1    1    b          :                              0.000000000000e+00
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.073486065232
 E-N energy              :            (au)                 -3.566631555860
 Coulomb energy          :            (au)                  1.328363917962
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.653870455401
 N-N energy              :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.818652028067
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.104512742098e+00
                         :      (kcal/mol)             -6.930922098896e+02
                         :        (kJ/mol)             -2.899897806178e+03
                         :            (eV)             -3.005532283883e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.345283968282
    1    1    b          :            (au)                 -0.345283968282
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.690567936564
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.000023
 Magnitude               :            (au)                        0.000023
                         :         (Debye)                        0.000059
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000023
 Magnitude               :            (au)                        0.000023
                         :         (Debye)                        0.000059
===========================================================================


===========================================================================
                      Magnetizability (mag-0.000000)
---------------------------------------------------------------------------
 Frequency               :            (au)                        0.000000
                         :          (cm-1)                        0.000000
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Diamagnetic             :       -0.913111        0.000000        0.000000
                         :        0.000000       -0.913111       -0.000000
                         :        0.000000       -0.000000       -0.772210
 Isotropic average       :            (au)                       -0.866144
                         :            (SI)                      -68.377861
---------------------------------------------------------------------------
 Paramagnetic            :       -0.000000       -0.000000       -0.000000
                         :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
 Isotropic average       :            (au)                             N/A
                         :            (SI)                             N/A
---------------------------------------------------------------------------
 Total tensor            :       -0.913111        0.000000        0.000000
                         :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
 Isotropic average       :            (au)                             N/A
                         :            (SI)                             N/A
===========================================================================


===========================================================================
                          NMR shielding (nmr-0h)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
 r_K                     :        0.000000        0.000000       -0.700144
---------------------------------------------------------------------------
 Diamagnetic             :       24.969077        0.000000       -0.000000
                         :        0.000000       24.969077        0.000000
                         :       -0.000000        0.000000       27.417679
---------------------------------------------------------------------------
 Paramagnetic            :       -0.000000       -0.000000       -0.000000
                         :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
---------------------------------------------------------------------------
 Total tensor            :       24.969077        0.000000       -0.000000
                         :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
---------------------------------------------------------------------------
 Diagonalized tensor     :             N/A             N/A             N/A
 Isotropic average       :           (ppm)                             N/A
 Anisotropy              :           (ppm)                             N/A
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 11s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
